src/notification/discord.ts

```diff
@@ -0,0 +1,30 @@
+import {Webhook, MessageBuilder} from 'discord-webhook-node';
+import {Config} from '../config';
+import {Logger} from '../logger';
+import {Link} from '../store/model';
+
+const hook = new Webhook(Config.notifications.discord.webHookUrl);
+const notifyGroup = Config.notifications.discord.notifyGroup;
+
+export function sendDiscordMessage(text: string, link: Link) {
+	(async () => {
+		try {
+			const embed = new MessageBuilder();
+			embed.setTitle('Stock Notification');
+			embed.addField('URL', text, true);
+			embed.addField('Brand', link.brand, true);
+			embed.addField('Model', link.model, true);
+
+			if (notifyGroup !== '') {
+				embed.addField('Attention', notifyGroup, true);
+			}
+
+			embed.setColor(65340);
```

Comment:
I believe this needs to be a hex string. I like this color :smile: `#58D68D`

Reply:
Sure I was just copying what they used to for the webhooks.info color, but I can change it to NVIDIA green #76b900

Reply:
Even better!